tests/snmpsim/nos3.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|Brocade-VDX-VCS 1 
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.9.1.1588
```

Comment:
The php unit tests are failing because this isn't matching. You are looking for `.1.3.6.1.4.1.1588.2.2.1.1` but providing `1.3.6.1.4.1.9.1.1588` so this test fails.